src/interfaces/dataset.interface.ts

```diff
@@ -0,0 +1,60 @@
+// SPDX-FileCopyrightText: 2024 PNED G.I.E.
+//
+// SPDX-License-Identifier: Apache-2.0
+export interface Dataset {
```

Comment:
I would create other types and pass them as dependencies of Dataset, such as publisher, contact...

Reply:
Good point. I will say lets merge this and i will open another PR to refactor this interface

Reply:
Fair enough